xmtp_mls/src/groups/message_history.rs

```diff
@@ -181,6 +199,74 @@ where
     }
 }
 
+#[allow(dead_code)]
```

Comment:
who is calling these functions? Is it the message history server directly?

Reply:
These functions will be called (in the next PR) by this library itself, as part of the message-history request-response cycle.
No - the message history server is just a dumb file server:  takes encrypted files in; sends encrypted files out, without ever knowing nor having any abilities to see(decrypt) the contents.

I think a flow diagram for how this feature-set works is in order.  Our partners will likely have the same question(s).